tensorflow_addons/activations/rrelu.py

```diff
@@ -14,18 +14,22 @@
 # ==============================================================================
 
 import tensorflow as tf
-from tensorflow_addons.utils.types import Number
-from tensorflow_addons.utils import types
-from typing import Optional
+from tensorflow_addons.utils.types import TensorLike, Number
+
+try:
+    tf.no_gradient("StatefulUniform")
+except:  # noqa: E722
+    pass
 
 
 @tf.keras.utils.register_keras_serializable(package="Addons")
 def rrelu(
-    x: types.TensorLike,
+    x: TensorLike,
     lower: Number = 0.125,
     upper: Number = 0.3333333333333333,
-    training: Optional[bool] = None,
-    seed: Optional[int] = None,
+    training: bool = None,
+    seed: Number = None,
+    gs: tf.random.Generator = None,
```

Comment:
Please use `Optional` here. Having None as the default value doesn't make the type optional automatically. I think it should but it doesn't :(